### DISCUSSION TITLE
A question about the static link when the application is compiled

### DISCUSSION THREAD

**Discussion by lizhi16** [unikraft/unikraft#457] [Q&A]
Hi, the Unikraft paper (Eurosys'21) clarifies that libs in the Unikraft are statically linked to the application (platform: linuxu, x86).

However, there are seems no linker flags like "-static" shown in the Makefile of Unikraft or this "Hello world" application. Could you please teach us how Unikraft achieves the static link in its Makefile? Thanks!

**Comment by razvand** [comment]
@lizhi16, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
